lars-reimann/api-editor
# feat: Package Parser: Explicit representation of parameter types and output as a joined property

Closes #488 .

### Summary of Changes

See #488


### Testing Instructions

Run `parse-package all -p sklearn -c <client_data> -o .\out\` 
Check the `type` attribute in `.\out\..._api.json` for the parameters
Check the `boundary` and `enum` annotations in `.\out\annotations.json`
